Developers/Git_Basics.rst

```diff
@@ -19,6 +19,13 @@ this will do:
 
     $ sudo apt-get install git-all
```

Comment:
I changed the short log. I guess it's fine now. Sorry for the delay, I was out of station.